### DISCUSSION TITLE
Can giscus be used with Furo theme?

### DISCUSSION THREAD

**Discussion by goroderickgo** [pradyunsg/furo#614] [Questions]
I've seen some [Sphinx themes](https://sphinx-immaterial.readthedocs.io/en/stable/customization.html#adding-a-comment-system) incorporate [giscus](https://giscus.app) as a commenting system. I'm relatively new to all of this, and I wasn't sure if this was something that required support by Furo. Thanks!

**Comment by pradyunsg** [comment]
This doesn't require theme support per-se.

You'll want to inject your own JS (and maybe HTML) into the page. See https://pradyunsg.me/furo/customisation/injecting/ for the details. :)

**Comment by goroderickgo** [comment]
@pradyunsg, sorry I am a noob when it comes to HTML and Sphinx themes. What I gathered so far is adding a block like this in a file called `_templates/page.html`, but as far as I can tell, it doesn't do anything. Would definitely appreciate any help you can provide

```html
{% extends '!page.html' %}

{% block body %}
    {{ super() }}
    <div>
        <script src="https://giscus.app/client.js"
                data-repo="e3-/kit"
                data-repo-id="MDEwOlJlcG9zaXRvcnkzMjkxMzIyNzQ="
                data-category="Documentation"
                data-category-id="DIC_kwDOE54o8s4CWsWE"
                data-mapping="pathname"
                data-strict="0"
                data-reactions-enabled="1"
                data-emit-metadata="0"
                data-input-position="top"
                data-theme="preferred_color_scheme"
                data-lang="en"
                data-loading="lazy"
                crossorigin="anonymous"
                async>
        </script>
    </div>
{% endblock %}
```
